modules/sapi_entity_interaction/sapi_entity_interaction.install

```diff
@@ -0,0 +1,9 @@
+<?php
+
+/**
+ * Implements hook_uninstall().
+ */
+function sapi_entity_interaction_uninstall() {
+  // Remove field_colors field on uninstall.
+  Drupal::configFactory()->getEditable('field.storage.node.field_colors')->delete();
+}
```

Comment:
I think is not meant to be here.  It is the only mention of field_colors in this code.